README.md

```diff
@@ -124,5 +128,16 @@ By using this tool you accept this risk and you take full responsiblity for the
     1. `make test`
     1. `make test-mqtt` - requires mosquitto MQTT broker binary
 
+## Hints for Bosswerk mi600 / mi300 to make them available for reading
+1. Open the hidden Menu of the inverter with http://<IP>/config_hide.html with Chrome (easiest with chrome)
+2. Mark the IP-Adress Field under "Internal Server parameters setting"
+    This Field normally has the AP-Adress 10.10.100.254 and is unable to edit
+3. Click right Mousebutton and investigate Field
+4. Delete the Text disabled="" from html-code
+5. Click right Mousebutton oder Element in HTML and Force Status ":active"
+6. Change IP to your internal inverter IP address
+7. Click Save AND DON'T REBOOT
+8. Wait some seconds and open http://<IP>/config_hide.html again to see if the changes had been saved. If not start again with 1
+9. If IP is changed, you can restart the inverter, changes will be permanent
     
```

Comment:
Thanks for this contribution. I think it's time to add a How-to page in the project. That would be a perfect place for this kind of tutorial. I will keep this parked until the How-to page is ready